Makefile

```
@@ -32,6 +35,5 @@ lint:
 	$(SELF) terraform/install terraform/get-modules terraform/get-plugins terraform/lint terraform/validate
 
 # Run acceptance tests
-testacc:
-	go get
+testacc: install
```

Should this be get ?

```suggestion
testacc: get
```

---

to test the terratest with the current provider code (not to download the provider from the registry),, we need to build and install it locally